base/applications/rapps/unattended.cpp

```diff
@@ -242,7 +242,10 @@ ParseCmdAndExecute(LPWSTR lpCmdLine, BOOL bIsFirstLaunch, int nCmdShow)
     {
         db.RemoveCached();
```

Comment:
I wonder whether this one needs to be done as well even in appwiz mode.

Reply:
Fixed.